cmd/sst/ui/ui.go

```diff
@@ -24,8 +24,9 @@ const (
 )
 
 const (
-	IconX     = "×"
+	IconX     = "x"
```

Comment:
lol we spent too much time picking this `x`, how come you wanna change it?

Reply:
@jayair it had issues in my terminal, sorry for hurting your feelings :p 
Also why tf are you people spending time a cross icon decision

Reply:
Oh really? Can I see a screenshot? I'm curious what it's doing.

Reply:
![image](https://github.com/sst/ion/assets/47459673/0af2b4dd-9e14-4941-8563-3700cd21b1da)

Reply:
maybe its my term sometimes it works sometimes it meses it up